config_operator/config_operator/config_source/LocalDirConfig.py

```diff
@@ -0,0 +1,68 @@
+import os
+import time
+import logging
+import yaml
+
+
+from config_operator.config_source.exceptions import UnreadableFileException
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+LISTEN_FOR_UPDATE_INTERVAL_SECONDS = 1
```

Comment:
Let's have this value be passed in to `__init__()` from `main()` instead of a being a top-level variable

Reply:
Ok, I did that. This is now available from the k8s operator. This looks nice !